winterfell/webapp/.eslintrc

```diff
@@ -8,5 +7,9 @@
         "_": true,
         "$": true,
         "angular": true
-    }
+    },
+    "rules": {
+      "no-console": "off"
```

Comment:
I was under the impression that this is enough, but it's still not ignoring console.log in our code :/